plugins/services/src/js/constants/ServiceErrorMessages.js

```diff
@@ -92,7 +92,12 @@ const ServiceErrorMessages = [
     )
   },
   {
-    path: /^environment\./,
+    path: /^environment\.[^.]+$/,
+    type: "GENERIC",
+    message: i18nMark("Must be valid")
```

Comment:
Not specific to this PR, but I just noticed that these are not getting translated (maybe because we change them to lower case?).
![Screenshot from 2019-06-13 11-18-44](https://user-images.githubusercontent.com/40791275/59415928-229c6e00-8dcd-11e9-8a2b-e253dd470869.png)
![Screenshot from 2019-06-13 11-19-27](https://user-images.githubusercontent.com/40791275/59415932-24fec800-8dcd-11e9-95b1-22032ad23b2b.png)